README.md

```diff
@@ -46,3 +46,23 @@ provider "maas" {
 ### Resources and Data Sources Configuration
 
 The [docs](/docs) section contains details about each supported Terraform resource and data source.
+
+### Release process
+
+1. Checkout main and create a new branch as `release-vX.X.X`
+   ```bash
+   git checkout main  
```

Comment:
Hmm, branch here is still called 'master'

We'd like to change it but don't want to break all your clones

Reply:
My mistake, fixed it.